client/src/components/SignUp/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { useState } from 'react';
```

Comment:
this is a super nit (don't take on this comment😂) but we could do 
```
import React, { useState } from 'react';
```
not that big of a deal at all tbh. but maybe for future components?

Reply:
Yeahh I much prefer it like that. For some reason TypeScript will auto import it into a separate import sometimes. I usually switch em out but I missed this guy.

Will fix and use my admin privileges to merge without re-review 💪